<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationModeControlTab</class>
 <widget class="QWidget" name="NavigationModeControlTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QRadioButton" name="standbyRadioButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Standby</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="navigateRadioButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Navigate</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="noOpRadioButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>No operation</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>applyButton</sender>
   <signal>pressed()</signal>
   <receiver>NavigationModeControlTab</receiver>
   <slot>applyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>applyPressed()</slot>
 </slots>
</ui>
